src/lib/routing-table.js

```diff
@@ -35,6 +35,28 @@ class RoutingTable {
     return false
   }
 
+  /**
+   * Compute a curve quote's `appliesToPrefix`, which is the shortest prefix
+   * that uniquely matches the target.
+   *
+   * @param {IlpAddress} routePrefix
+   * @param {IlpAddress} destinationAccount
+   * @returns {IlpAddress}
+   */
+  getAppliesToPrefix (routePrefix, destinationAccount) {
```

Comment:
I don't understand the need for this function. When a quote request comes in, the connector will check its routing table for the longest entry in its PrefixMap that matches. If this route has a curve, it returns that, setting `appliesToPrefix` in the response to the `destinationLedger` of the route used. If the route has no curve, it resorts to remote quoting, and gets `appliesToPrefix` from the next connector's response, right?